pkg/session_recording/recordings.jsx

```diff
@@ -433,7 +433,7 @@
                                             <button title="Play/Pause - Hotkey: p" type="button" ref="playbtn"
                                                 className="btn btn-default btn-lg margin-right-btn play_btn"
                                                 onClick={this.playPauseToggle}>
-                                                <i className={"fa fa-"+ this.state.playback} aria-hidden="true"></i>
+                                                <i className={"fa fa-"+ playback_state} aria-hidden="true"></i>
```

Comment:
This should be in the previous commit, IIUC. Anyway, if we switch to the boolean `this.state.paused`, this can become just a ternary operator: `(this.state.paused ? "play" : "pause")`.